### ISSUE TITLE
collect data from the CARLA leaderboard benchmark

### ISSUE THREAD

**Issue by a1wj1** [opendilab/InterFuser#48]
Thank you. May I ask how to modify the content in leaderboard/scripts/run_evaluation.sh if I want to collect data from the CARLA leaderboard benchmark?

![1688436601887](https://github.com/opendilab/InterFuser/assets/38905689/8b801a33-4dfe-46ac-a514-7d582ff8191c)

Is it correct to only modify SCENARIOS and ROUTES？as shown in the figure.

**Comment by deepcs233** [comment]
Yes, you are right!

**Comment by deepcs233** [comment]
The more the better, `routes_training.xml` has the most proportion of the data.

**Comment by a1wj1** [comment]
According to the indicators in your paper, did you run three xmls together？

**Comment by deepcs233** [comment]
No, we collect the drivinf data based on other xmls (please refer to repo#data collection)
